.github/workflows/scripts/comments.mjs

```diff
@@ -1,12 +1,71 @@
+import { readFile } from 'node:fs/promises'
+
+const DIST_DIFFS = [
+  {
+    file: 'dist-js.diff',
+    title: 'JavaScript changes to `dist`',
+    marker: 'dist-diff-js'
+  },
+  {
+    file: 'dist-css.diff',
+    title: 'CSS changes to `dist`',
```

Comment:
Minor thing, but we seem to say styles or "Stylesheets" to match scripts or "JavaScripts"